apps/src/applab/applab.js

```diff
@@ -647,9 +641,9 @@ Applab.init = function (config) {
   }
 
   // If we are in mobile sharing mode, allow the viewport to handle scaling
-  // and override our default width target in vizAppWidth with the actual width
+  // and override our default width target in VIZ_APP_WIDTH with the actual width
   if (dom.isMobile() && config.hideSource) {
-    vizAppWidth = Applab.appWidth;
+    VIZ_APP_WIDTH = Applab.appWidth;
```

Comment:
This part is a little odd. The reason that we weren't using all caps `VIZ_APP_WIDTH` is because we do actually change this value under some circumstances. Reading it this way makes me a little uncomfortable from a style perspective, because it looks like a constant, but it's not.